apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/shareEmbedModal/anonymous-links-tab.test.tsx

```
@@ -0,0 +1,389 @@
+  ShareSurveyLink: ({ surveyUrl, publicDomain }: any) => (
+    <div data-testid="share-survey-link">
+      <p>Survey URL: {surveyUrl}</p>
+      <p>Public Domain: {publicDomain}</p>
+    </div>
+  ),
+}));
+
+vi.mock("@/modules/ui/components/advanced-option-toggle", () => ({
+  AdvancedOptionToggle: ({ children, htmlId, isChecked, onToggle, title, description }: any) => (
+    <div data-testid={`toggle-${htmlId}`} data-checked={isChecked}>
+      <button data-testid={`toggle-button-${htmlId}`} onClick={() => onToggle(!isChecked)}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)